requirements.txt

```diff
@@ -0,0 +1,8 @@
+calendar
+re
+sys
+abc
+datetime
```

Comment:
these are built-in libraries ... I believe we have them in the mac environment too ?